azure-pipelines.release.yml

```diff
@@ -23,173 +24,188 @@ schedules:
-      publishLocation: 'Container'
-    displayName: 'Publish artifact: package-manifest'
-
-  - task: PublishBuildArtifacts@1
-    inputs:
-      pathtoPublish: packages/fluentui/docs/src/public
-      artifactName: 'demo-images'
-      publishLocation: 'Container'
-    displayName: 'Publish artifact: demo-images'
-
-  # Run this near the end because it's more likely to fail than the artifact upload tasks, and its
-  # failure doesn't need to block anything else
-  - script: |
-      node -r ./scripts/ts-node-register ./scripts/updateReleaseNotes/index.ts --token=$(githubPAT) --apply --debug
-    displayName: 'Update github release notes'
-
-  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
-    displayName: 📒 Generate Manifest
-    inputs:
-      BuildDropPath: $(System.DefaultWorkingDirectory)
-
-  - task: PublishPipelineArtifact@1
-    displayName: 📒 Publish Manifest
-    inputs:
-      artifactName: SBom-$(System.JobAttempt)
-      targetPath: $(System.DefaultWorkingDirectory)/_manifest
-
-  # This would usually be run automatically (via a pipeline decorator from an extension), but the
-  # thorough cleanup step prevents it from working. So run it manually here.
-  - task: ComponentGovernanceComponentDetection@0
-    displayName: 'Component Detection'
-    inputs:
-      sourceScanPath: $(Agent.BuildDirectory)
-    condition: succeeded()
-    timeoutInMinutes: 5
-    continueOnError: true
-
-  - template: .devops/templates/cleanup.yml
-    parameters:
-      checkForModifiedFiles: false
+jobs:
+  - template: .devops/templates/compliance-job.yml
+
+  - job: Release
+    dependsOn: Compliance
+    pool: '1ES-Host-Ubuntu'
+    workspace:
+      clean: all
+    steps:
+      - template: .devops/templates/tools.yml
+
+      - script: |
+          git config user.name "Fluent UI Build"
+          git config user.email "fluentui-internal@service.microsoft.com"
+          git remote set-url origin https://github.com/microsoft/fluentui.git
+        displayName: Authenticate git for pushes
+    #      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/fluentui.git
+
+
+      - task: Bash@3
+        inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      - script: |
+          yarn generate-version-files
+        displayName: Generate version files
+
+      - script: |
+          yarn run:published build --production
+        displayName: yarn build
+
+      # --only makes it only run tests (otherwise due to the missing --production arg, lage would re-run the build)
+      # https://github.com/microsoft/fluentui/issues/21686
+      - script: |
+          yarn run:published test --only
+        displayName: yarn [test]
+
+      - script: |
+          yarn run:published lint --only
+        displayName: yarn lint
+
+      - script: |
+          yarn run:published bundle --only --production
+        displayName: yarn bundle
+
+      # - script: |
```

Comment:
can we remove comments that have no purpose  ?

Reply:
missed reverting...

Reply:
and same for one lower. Fixed that one too. Thanks for catching!